app/src/test/java/com/locoquest/app/BenchmarkUnitTests.kt

```diff
@@ -58,8 +61,8 @@ class BenchmarkServiceTest {
                 elevation = 8848.86,
                 description = 0.0,
                 d = 0.0,
-                d1 = 0.0
-            )
```

Comment:
Originally the unit test errored out here and did not Parse data. Attempted reading this data in as an Array, but I was not able to get this to parse properly